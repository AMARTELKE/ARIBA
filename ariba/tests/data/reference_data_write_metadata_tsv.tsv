gene2	1	0	.	.	we didn't burn him
gene1	1	0	.	.	has anybody got a bottle of orange juice?
