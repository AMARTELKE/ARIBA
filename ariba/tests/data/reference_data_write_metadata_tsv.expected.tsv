gene1	.	.	has anybody got a bottle of orange juice?
gene2	.	.	we didn't burn him
